src/services/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './basic-desc/basic-desc.service';
```

Comment:
I'll remove the layer services, but basic-desc and basic-static-primitives are extensible this way.